api/app/GraphQL/Mutations/PlaceCandidate.php

```diff
@@ -25,6 +26,12 @@ public function __invoke($_, array $args)
         $candidate->computed_final_decision = $finalDecision['decision'];
         $candidate->computed_final_decision_weight = $finalDecision['weight'];
 
+        if ($placementType === PlacementType::PLACED_TERM->name || $placementType === PlacementType::PLACED_INDETERMINATE->name) {
```

Comment:
Could you add a descriptive comment to this block, please?

Reply:
Okay, done in https://github.com/GCTC-NTGC/gc-digital-talent/pull/12330/commits/46ff561c360543acb0c76fc4cbe1b22bd710d28f